packages/ibm-products/src/components/CoachmarkOverlayElements/_storybook-styles.scss

```
@@ -0,0 +1,20 @@
+//
+// Copyright IBM Corp. 2023, 2023
```

You might imagine that at this point, I do regret my commitment to making these easy to batch commit. 😹 😭 

```suggestion
// Copyright IBM Corp. 2023, 2024
```